packages/manager/apps/hpc-vmware-managed-vcd/src/components/message/MessageList.component.tsx

```diff
@@ -2,11 +2,13 @@ import React from 'react';
 import { useMessageContext } from '@/context/Message.context';
 import { Message } from './Message.component';
 
-export const MessageList: React.FC = () => {
+type Props = { className?: string };
+
+export const MessageList: React.FC<Props> = ({ className }) => {
   const { messages } = useMessageContext();
 
   return (
-    <div className="flex flex-col">
+    <div className={`flex flex-col ${className}`}>
```

Comment:
For information I think if `className` is `undefined`. You can see the class `undefined` applied. It's not very important but I prefer `${className ?? ''}`to avoid this effect